package.json

```diff
@@ -1923,7 +1923,7 @@
 				},
 				{
 					"command": "openshift.resource.watchLogs",
-					"when": "view == openshiftProjectExplorer && viewItem =~ /^openshift\\.k8sObject\\.(?!helm)/"
+					"when": "view == openshiftProjectExplorer && viewItem =~ /^openshift\\.k8sObject\\.!(helm&&pod)/"
```

Comment:
Unfortunately after this change, the watch logs and open in developer console context menus are always hidden.

I think the correct regex is `/^openshift\\.k8sObject\\.(?!helm|pod)/`

Please check out https://www.rexegg.com/ and https://regex101.com/ for working with regex